docker/docker-compose.yml

```diff
@@ -15,82 +15,27 @@ services:
         - RPCUSER
         - RPCPASS
         - NETWORK
+        - DEBUG
+        - MINING_ADDRESS
+      entrypoint: ["./start-btcd.sh"]
 
-    btcd:
-        extends: btc
-        container_name: btcd
-        environment:
-          - DEBUG
-          - MINING_ADDRESS
-          - NETWORK
-        entrypoint: ["./start-btcd.sh"]
-
-    btcctl:
-        extends: btc
-        container_name: btcctl
-        links:
-            - "btcd:rpcserver"
-        entrypoint: ["./start-btcctl.sh"]
-
-
-    # ltc is an image of litecoin node which used as base image for ltcd and
-    # ltcctl. The environment variables default values determined on stage of
-    # container start within starting script.
-    ltc:
-      image: ltcd
+    lnd:
+      image: lnd
+      container_name: lnd
       build:
-        context: ltcd/
-      volumes:
-        - shared:/rpc
-        - litecoin:/data
+        context: ../
+        dockerfile: docker/lnd/Dockerfile
       environment:
         - RPCUSER
         - RPCPASS
         - NETWORK
-
-    ltcd:
-        extends: ltc
-        container_name: ltcd
-        environment:
-          - DEBUG
-          - MINING_ADDRESS
-          - NETWORK
-        entrypoint: ["./start-ltcd.sh"]
-
-    ltcctl:
-        extends: ltc
-        container_name: ltcctl
-        links:
-            - "ltcd:rpcserver"
-        entrypoint: ["./start-ltcctl.sh"]
-
-    lnd:
-        image: lnd
-        build:
-          context: ../
-          dockerfile: docker/lnd/Dockerfile
-        environment:
-          - RPCUSER
-          - RPCPASS
-          - NETWORK
-          - CHAIN
-          - DEBUG
-        volumes:
-          - shared:/rpc
-          - lnd:/root/.lnd
-        entrypoint: ["./start-lnd.sh"]
-
-    lnd_ltc:
-      extends: lnd
-      container_name: lnd_ltc
-      links:
-          - "ltcd:blockchain"
-
-    lnd_btc:
-      extends: lnd
-      container_name: lnd_btc
+        - CHAIN
+        - DEBUG
+      volumes:
+        - shared:/rpc
```

Comment:
You removed the persistent volume for the lnd data here, which probably happened on rebase.
Please add `          - lnd:/root/.lnd` again.

Reply:
good catch :fish: